kitchen.docker.recipes.yml

```diff
@@ -0,0 +1,39 @@
+# Allows to test single recipes which can be tested on Docker.
```

Comment:
Shouldn't this file be decoupled from the driver (`kitchen.recipes.yml` with no `docker` in the name)?

Reply:
Done